src/GrpcProtos/services.proto

```diff
@@ -98,7 +98,6 @@ service FramePointerValidatorService {
 message CrashOrbitServiceRequest {
   enum CrashType {
     CHECK_FALSE = 0;
-    NULL_POINTER_DEREFERENCE = 1;
```

Comment:
Is there some best-practice with removing enum values here? Should we mark the id as reserved or something?

Reply:
Yes, deleting from a proto hurt a bit when typing it. But since we do not care about staying compatible with different versions of service <-> client we don't care. And this thing is never serialized.

However we can keep the field and deprecate it which would be the more robust version of handling this. I'll bring this up in a standup.

Reply:
"deprecated = true" did not work for unclear reason, prefixing the member with OBSOLETE_ is another way to mark deprecated members.

Reply:
Thanks. SGTM!